ddtrace/contrib/trace_utils.py

```diff
@@ -595,3 +597,54 @@ def set_user(tracer, user_id, name=None, email=None, scope=None, role=None, sess
             "See https://docs.datadoghq.com/security_platform/application_security/setup_and_configure/"
             "?tab=set_user&code-lang=python for more information.",
         )
+
+
+def _start_span_internal(
```

Comment:
nit: I think this name is more readable
```suggestion
def _start_internal_span(
```